FilePersistence/src/version_control/merge/ListMergeComponentV2.cpp

```diff
@@ -25,6 +25,13 @@
 ***********************************************************************************************************************/
 
 #include "ListMergeComponentV2.h"
+#include "../../filepersistence_api.h"
+#include "MergeChange.h"
+
+#include "ModelBase/src/persistence/PersistentStore.h"
```

Comment:
Please, always put includes from other plug-ins after includes of the current plug-in. Also group gether includes from the same plug-in.